crates/astria-bridge-withdrawer/ethereum/out/AstriaWithdrawer.sol/AstriaWithdrawer.json

```diff
@@ -0,0 +1 @@
3:3:0","typeDescriptions":{"typeIdentifier":"t_magic_message","typeString":"msg"}},"id":16,"isConstant":false,"isLValue":false,"isPure":false,"lValueRequested":false,"memberLocation":"277:6:0","memberName":"sender","nodeType":"MemberAccess","src":"273:10:0","typeDescriptions":{"typeIdentifier":"t_address","typeString":"address"}},{"expression":{"id":17,"name":"msg","nodeType":"Identifier","overloadedDeclarations":[],"referencedDeclaration":-15,"src":"285:3:0","typeDescriptions":{"typeIdentifier":"t_magic_message","typeString":"msg"}},"id":18,"isConstant":false,"isLValue":false,"isPure":false,"lValueRequested":false,"memberLocation":"289:5:0","memberName":"value","nodeType":"MemberAccess","src":"285:9:0","typeDescriptions":{"typeIdentifier":"t_uint256","typeString":"uint256"}},{"id":19,"name":"memo","nodeType":"Identifier","overloadedDeclarations":[],"referencedDeclaration":11,"src":"296:4:0","typeDescriptions":{"typeIdentifier":"t_bytes_calldata_ptr","typeString":"bytes calldata"}}],"expression":{"argumentTypes":[{"typeIdentifier":"t_address","typeString":"address"},{"typeIdentifier":"t_uint256","typeString":"uint256"},{"typeIdentifier":"t_bytes_calldata_ptr","typeString":"bytes calldata"}],"id":14,"name":"Withdrawal","nodeType":"Identifier","overloadedDeclarations":[],"referencedDeclaration":9,"src":"262:10:0","typeDescriptions":{"typeIdentifier":"t_function_event_nonpayable$_t_address_$_t_uint256_$_t_bytes_memory_ptr_$returns$__$","typeString":"function (address,uint256,bytes memory)"}},"id":20,"isConstant":false,"isLValue":false,"isPure":false,"kind":"functionCall","lValueRequested":false,"nameLocations":[],"names":[],"nodeType":"FunctionCall","src":"262:39:0","tryCall":false,"typeDescriptions":{"typeIdentifier":"t_tuple$__$","typeString":"tuple()"}},"id":21,"nodeType":"EmitStatement","src":"257:44:0"}]},"functionSelector":"0968f264","implemented":true,"kind":"function","modifiers":[],"name":"withdraw","nameLocation":"200:8:0","parameters":{"id":12,"nodeType":"ParameterList","parameters":[{"constant":false,"id":11,"mutability":"mutable","name":"memo","nameLocation":"224:4:0","nodeType":"VariableDeclaration","scope":23,"src":"209:19:0","stateVariable":false,"storageLocation":"calldata","typeDescriptions":{"typeIdentifier":"t_bytes_calldata_ptr","typeString":"bytes"},"typeName":{"id":10,"name":"bytes","nodeType":"ElementaryTypeName","src":"209:5:0","typeDescriptions":{"typeIdentifier":"t_bytes_storage_ptr","typeString":"bytes"}},"visibility":"internal"}],"src":"208:21:0"},"returnParameters":{"id":13,"nodeType":"ParameterList","parameters":[],"src":"247:0:0"},"scope":24,"stateMutability":"payable","virtual":false,"visibility":"external"}],"abstract":false,"baseContracts":[],"canonicalName":"AstriaWithdrawer","contractDependencies":[],"contractKind":"contract","fullyImplemented":true,"linearizedBaseContracts":[24],"name":"AstriaWithdrawer","nameLocation":"81:16:0","scope":25,"usedErrors":[],"usedEvents":[9]}],"license":"MIT or Apache-2.0"},"id":0}
```

Comment:
i committed this because otherwise we have to install foundry in ci which was painful last time - however this will need to be updated if the contract changes. thoughts?

Reply:
i agree - its nicer to have these committed rather than having to deal with with foundry every time.

Reply:
Absolutely agreed here